guides/promotion.rst

```diff
@@ -182,8 +182,139 @@ Co lze průběžně propagovat
+
+.. note::
+   Používání zkratky *CfP* nebo sousloví *Call for Papers* celkem funguje v zahraničí, ale Češi a Slováci zatím moc nerozumí ani tomuto označení, ani principu, který označuje. Je dobré se tedy při tuzemské komunikaci těmto názvům vyhnout a princip vysvětlovat. Místo "CfP is open" psát "Now you can submit your talk or workshop proposal", apod.
+
+Plošné bombardování
+^^^^^^^^^^^^^^^^^^^
+
+Když je otevřeno CfP, je dobré to dát plošně vědět, protože existuje pár lidí, kterým toto oznámení stačí a následně něco do CfP pošlou. Není dobré se tím ale vyčerpávat, protože zas tak moc těch lidí není. Nezapomínat na e-mailové skupiny jako `django-cs <https://groups.google.com/group/django-cs/>`__ a `konference py.cz <https://www.py.cz/mailman/listinfo/python>`__, kde jsou také lidé, kteří na toto mohou reagovat.
+
+Agregátory na Twitteru
+^^^^^^^^^^^^^^^^^^^^^^
+
+Může se vyplatit oslovit různé agregátory, aby náš tweet retweetli a tak se dostal k více lidem, např. `@callbackwomen <https://twitter.com/callbackwomen>`__. Z osobního účtu jim můžeme jen tak poslat `tweet <https://twitter.com/honzajavorek/status/963867692594638848>`__ s odkazem na konferenční tweet, který bychom rádi zpropagovali.
+
+.. _priprava-na-cfp:
+
+Příprava na CfP
+^^^^^^^^^^^^^^^
+
+Před CfP je dobré si udělat "domácí úkoly". Stejně jako si sportovní komentátoři před zápasem Japonska s Guineí Bissau musí nastudovat hráče a jejich reálie, aby měli o čem celý zápas mluvit, může se i Media tým složený z neprogramátorů připravit na propagaci CfP.
+
+Základem je vědět obecně koho chci do CfP dostat. PyCon CZ je historicky hodně o tom, že ukazuje diverzitu světa Pythonu. Zajímavé je tedy dostat do CfP lidi, kteří nejsou tak úplně programátoři (`biologové <https://cz.pycon.org/2018/programme/detail/talk/21/>`__, datoví novináři, `fyzikové <https://cz.pycon.org/2017/speakers/detail/talk/22/>`__, ...), začátečníky, ženy, `děti <https://cz.pycon.org/2018/programme/detail/talk/33/>`__, atd. Takže pokud někde někoho takového zahlédnu, chňapnu po něm a budu se jej snažit do CfP dotlačit.
+
+Potom PyCon CZ mívá expertní přednášky, které nejsou zajímavé tím, že jsou z (pro programátory) obskurního prostředí nebo že je přednáší `desetiletý chlapec <https://twitter.com/all_about_code>`__, ale jsou zajímavé prostě tématem. V tomto případě jsou samozřejmě nějaké evergreen témata, která letí vždycky (databáze, API, komunita, vzdělávání, ...), ale pak je dobré i zjistit, co zrovna letí.
```

Comment:
"nějaká evergreen témata"

Reply:
k tomu "ale pak je dobré i zjistit, co zrovna letí" - vzhledem k návaznosti na předchozí větu bych to trochu přeformulovala, aby tam bylo trochu víc zdůrazněné to, že to je pro aktuální období, v kontrastu s tím, co letí vždy. Něco jako "ale pak je dobré i zjistit, co letí právě teď." nebo "ale pak je dobré i zjistit, jaká jsou aktuální populární témata." nebo tak něco...